src/Microsoft.OData.Client/ALinq/ApplyQueryOptionExpression.cs

```diff
@@ -0,0 +1,80 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ApplyQueryOptionExpression.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Client
+{
+    using System;

```

Comment:
Indentation

Reply:
@wachugamaina Is there an issue with indentation here? The usings when placed within the namespace should be indented

Reply:
@wachugamaina Traced the indentation issue and fixed